src/config.js

```diff
@@ -0,0 +1,33 @@
+define(['lodash-amd/modern/objects/defaults',], function (defaults) {
+
+  var defaultOptions = {
+    allowBlockElements: true,
+    debug: false,
+    undo: {
+      manager: false,
+      enabled: true,
+      limit: 100,
+      interval: 250
+    },
+    defaultCommandPatches: [
+      'bold',
+      'indent',
+      'insertHTML',
+      'insertList',
+      'outdent',
+      'createLink'
+    ]
+  };
+
+
+  function checkOptions(userSuppliedOptions) {
+    var options = userSuppliedOptions || {};
+
+    return Object.freeze(defaults(options, defaultOptions));
```

Comment:
Correct me if I am wrong, but with this line you intend on freezing the options after scribe initialization. While this might be a good programming practice, I am afraid it will bring back the FF31/32 `test/undo-manager.spec.js` issue back.